Guidance (#62)
alan-turing-institute/clim-recal: README.md

# Welcome to the `clim-recal` repository! 

> Welcome to clim-recal, a specialized resource designed to tackle systematic errors or biases in Regional Climate Models (RCMs). As researchers, policy-makers, and various stakeholders explore publicly available RCMs, they need to consider the challenge of biases that can affect the accurate representation of climate change signals. Clim-recal provides both a **broad review** of available bias correction methods as well as **practical tutorials** and **guidance** on how to easily apply those methods to various datasets.

Reviewer: ```suggestion
Welcome to clim-recal, a specialized resource designed to tackle systematic errors or biases in Regional Climate Models (RCMs). As researchers, policy-makers, and various stakeholders explore publicly available RCMs, they need to consider the challenge of biases that can affect the accurate representation of climate change signals. 

Clim-recal provides both a **broad review** of available bias correction methods as well as **software**, **practical tutorials** and **guidance** that helps users apply these methods methods to various datasets.
```

Clim-recal is an **Extensive guide to application of BC methods**: 

- Accessible information about the [why and how of bias correction for climate data](#why-bias-correction)

> - Technical resource for application BC methods (see our full pipeline for bias-correction of the ground-breaking local-scale (2.2km)[Convection Permitting Model (CPM)](https://www.metoffice.gov.uk/pub/data/weather/uk/ukcp18/science-reports/UKCP-Convection-permitting-model-projections-report.pdf))

Reviewer: ```suggestion
- Is a software library for for the application of BC methods (see our full pipeline for bias-correction of the ground-breaking local-scale (2.2km)[Convection Permitting Model (CPM)](https://www.metoffice.gov.uk/pub/data/weather/uk/ukcp18/science-reports/UKCP-Convection-permitting-model-projections-report.pdf)). Clim-recal brings together different software packages in Python and R that implement a variety of bias correction methods, making it easy to apply them to data and compare their outputs.
```

> - In partnership with the MetOffice to ensure the propriety, quality, and usability of our work

Reviewer: ```suggestion
- Was developed in partnership with the MetOffice to ensure the propriety, quality, and usability of our work
```

> - Framework for open additions (in planning)

Reviewer: ```suggestion
- Provides a framework for open additions of new software libraries/bias correction methods (in planning)
```
